### ISSUE TITLE
Can't open config

### ISSUE THREAD

**Issue by StevenRenders** [cactusss/nl.onzewifi.insights2sql#3]
Noticed today that I can't update the config:

SQL tab remains empty
Log tab remains empty
Device tab initially shows the current list but whenever I select a device, it also turns blank.
I really need to add some devices. I'm loosing data every minute...

Tried app restart and Homey reboot.
Homey 2023 vers. 10.1.0
App 1.7.2

Diagnostics: 138e7aa3-58e5-4e25-9ad3-4d7863ee08a5
Issue occurs in app, on webpage, on developer site.

**Comment by StevenRenders** [comment]
Also noticed now that since a couple of days the cpu usage has increased significantly.
![image](https://github.com/cactusss/nl.onzewifi.insights2sql/assets/77631462/8db88f76-46d6-470e-b6a9-f77aaa29c527)

**Comment by matswhansen** [comment]
Same here, after migrating from HP19 to HP23

**Comment by Boechie** [comment]
_I originally left this comment here (https://github.com/cactusss/nl.onzewifi.DeebotT9/issues/9) as the Homey store links to report any issues there. It seems inaccurate. I think it should link to this repository however. Therefore again:_

I'm also experiencing the same. I can only see the list of devices to select for saving insights. As soon as I click anything (another tab, or a device) a blank page is shown. This is in the Homey web-app as well as in the Android mobile app. This worked before. Today I noticed this for the first time. Last time a checked was a few weeks ago and then it worked as it should.

**This issue is really a blocker!** It is now impossible to make any config changes, so not even make a DB connection, let alone log something to a database. I noticed however that settings that I made before are respected. I still get logs of all insights that I set up earlier.

@cactusss Could you please solve this issue? If you need any help, please let me know and I'll ask our development team to help you out.

The problem exists in my HP23 enviroments (v 10.2.1) as well as on my HP19 environment (v 10.0.4).

**Comment by StevenRenders** [comment]
There seems to be no response from @cactusss on this whatsoever.

This becomes a big showstopper for my data warehouse! Loosing data every day :-( 

Is there any way we can debug this ourselves with the sourcecode at hand?

**Comment by Boechie** [comment]
@StevenRenders I agree, this is really a blocker and renders the Insights2Sql app totally useless. Happily in my case things that worked are still working, but I can't create or change anything to the current logging setup. Of course you can try to debug. The code is in this repository, but it requires quite some programming knowledge and time.

After a first quick glance I suspect that the problem is somewhere with the Vue.js component that is being used for the settings. Maybe this component had an update (guessing). The code of this Vue.js component is loaded at runtime from their servers and not part of the Insights2Sql package. Or it is this Vue.js component in combination with the (very crappy) updated version of the Homey software. There were no recent changes to the Insights2Sql app and although Homey has major problems after the update last week (Jan 9th) I don't see a relation of this issue with all the other issues in other apps after the Homey update.

Although debugging and fixing by yourself is an option, we're still dependant on @cactusss to approve and merge the PR that anyone would create with a fix. It would be really helpful is @cactusss would reply. I already offered help from my development team, but if @cactusss stays MIA it is pointless to waste time on a fix that we can not approve/merge.

@cactusss, could you **please** look into this? I'm offering help but this can only be done with your help. Please at least give us a reply. Thank you very much!

**Comment by matswhansen** [comment]
As a workaround until Insight2Sql is done, if possible: Setup a MySql app and write important stuff there, and then create a linked server and copy the data to your SQL server every hour :)

**Comment by cactusss** [comment]
The problem is solved with versie 1.7.3 (https://homey.app/nl-nl/app/nl.onzewifi.insights2sql/&lt;-Insights2SQL%C2%B2-&gt;/)

**Comment by Boechie** [comment]
Very big **thank you** to @cactusss !

**Comment by StevenRenders** [comment]
Also from me, Thanks @cactusss for solving this, you saved the day!
